### DISCUSSION TITLE
Zur Moralität und Ethik von SSI

### DISCUSSION THREAD

**Discussion by hairmare** [e-id-admin/governance-sounding-board#22] [General]
Ich habe mich in den letzten Wochen etwas näher mit dem Begriff Self Sovereign Identity (deutsch: Selbstbestimmte Identität, kurz: SSI) auseinander gesetzt und möchte meine Gedanken dazu hier teilen.

In der Deutschen Wikipedia [finden wir](https://de.wikipedia.org/wiki/Selbstbestimmte_Identit%C3%A4t) im Bezug auf auf die Schweiz folgende Aussage:

> Der [Bundesrat](https://de.wikipedia.org/wiki/Bundesrat_(Schweiz)) beschloss im Dezember 2021, eine staatliche elektronische Identität (E-ID) auf der Basis der SSI zu schaffen.

Ich hege Zweifel, ob diese Aussage so der Absicht des Bundesrates entspricht und begrüsse denn auch, dass in den Folien vom ersten Partizipations-Meeting SSI nur als eine unter vielen Lösungen, die es zu erwägen gilt, kommuniziert wurde.

Ich denke bei der Beurteilung von SSI als Basis von digitialer Indentifikation gibt es einige moralische und ethische Fragen zu berücksichtigen. Eine Abhandlung zu diesen Fragen findet sich z.Bsp. im Paper "Sovereignty, privacy, and ethics in blockchain-based identity management systems" von G. Ishmaev: https://link.springer.com/article/10.1007/s10676-020-09563-x

Das Paper zeigt insbesondere auf, dass es imperativ ist, zur Umsetzung von SSI einen moraltheoretischen Rahmen zu schaffen der verhindert, dass damit die Büchse der Pandora geföffnet wird.

Persönlich glaube ich nicht, dass dieser Rahmen seit der Publikation des Papers im November 2020 geschaffen wurde. Ich denke es ist noch einiges an Grundlagenforschung notwendig um ebendiesen Rahmen zu schaffen. Diese Notwendigkeit sehe ich in einem klaren Zielkonflikt mit den akuellen E-ID Plänen.

Bereits seit Jahren versuchen wir in der Schweiz eine Digitale ID zu etablieren und spätestens seit der Ablehnung des Bundesgesetzes über elektronische Identifizierungsdienste in der Volksabstimmung vom 7. März 2021 ist klar, dass uns diese Aufgabe noch eine Weile beschäftigen wird. Um ein neues E-ID Gesetz Mitte 2022 in die Vernehmlassung zu schicken, fehlt schlicht die Zeit um, den in dem Paper gestellten Herausforderungen, mit der notwendigen Sorgfalt Rechnung zu tragen.

**Comment by grt-dti** [comment]
mir war nicht klar, wie strukturiert das Problem der Technokratie von blockchainbasierten Lösungen in der Wissenschaft angegangen wird. Wirklich brauchbare Antworten werden zwar nicht geliefert, aber das Problem zumindest eingehend beleuchtet.
Ein Hinweis vielleicht bereits vorab: Das BGEID (also das kommende) wird nach Aussagen des Projektteams technologieagnostisch sein - es wird also keine Vorgaben zu Blockchains & Co., ja wahrscheinlich nicht einmal zu SSI als Technologie machen, sondern viel mehr die Ideen von SSI als Sammlung von Prinzipien zur Bewirtschaftung digitaler Identitäten normativ festlegen. Nach allem, was ich aus dem ersten Artikel des Paper rauslesen konnte, wird dies aber vielleicht dennoch zu einem Kredit für die allfällige Volksabstimmung werden, weil SSI effektiv nur mittels einer Blockchain umfänglich umgesetzt werden kann - SSI schreit quasi nach Blockchain und damit "schreit" es auch nach den im Paper aufgezeigten Problemen.....

**Comment by romanzoun** [comment]
Hoi

ich glaube man muss bei den verifiable credentials, also den unterschiedlichen digitalen Nachweise einen "Level of Assurance", also einen Sicherheitslevel definieren, welcher die Mindestanforderung an die Nutzung prägt. Also hoher Sicherheitslevel, also eID, Geburtsurkunde etc, erfordert Trust anchor für Verifier und Issuer, Biometrischen Schutz der Wallet und am besten noch 2FA und Video-Ident. Mittlerer Level, wie Passwordless eBanking Login, nur 2FA und Trust Anchor und der niedrigste Level, wäre mein fake twitter account und mein Account für ein Online spiel, die kann ich einfach so rum zeigen und bin in Verantwortung diese Daten zu schützen.

Klarer Gewinn auf allen Seiten --> keine Passwörter, hohe Sicherheit, digitale verifizierte Identität für Services

Blockchain ist kein Muss bei SSI. Bei SSI ist es wichtig eine "dezentrale" konsistente Datenbank zu haben, wo nur bestimmte Leute schreiben dürfen (Issuer), Das wird vom vertrauenswürdigen Anbieter (Staat) gemanaged. Der Nutzer hält die Daten in seiner Wallet, er muss jedes Mal je nach Sicherheitslevel die Datenfreigabe bestätigen, keiner kriegt die Transaction ausser der Holder und der genutzte Service.

**Ich sehe keine ethische Bedenken**, denn auf der dezentralen Datenbank, steht nichts geheimes drin. Das Dezentrale, also Blockchain verringert Ausfallrisiko und erschwert die Trackbarkeit der Transaktionen. Die Infrastruktur, also Blockchain kann auch durch eine Zentrales Register ersetzt werden, das eröffnet dem Betreiber eine Art Trackbarkeit der Nutzer....Aber wenn es der Staat macht, dann ist es in meinen Augen ok.

Ich freue mich auf einen Austausch was Ethik und Moral von SSI angeht :)

Cheers, Roman

**Reply by grt-dti** [reply]
Zustimmung und Ablehnung zugleich (dafür gibts kein Emoji).
Zustimmung für die differenzierenden LoA's und die Positionierung des E-ID-Ausweises auf dem höchsten Level.
Ablehnung bei der Annahme "Blockchain ist kein Muss bei SSI". Doch, leider ist das so. Die SSI-Prinzipien (man könnte es auch als Wirkungsversprechen der SSI interpretieren) kann man nur mit einer Blockchain in ihren maximalen Ausprägungen erfüllen/erreichen und nichts und niemand verhindert die Forderung nach der "maximalen Ausprägung der SSI-Versprechen" (warum auch, dient es doch der maximalen Erfüllung eines individualisiert betrachteten moralisch integren Nutzervorteils) . Am Ende kommen wir schlicht bei einer blockchainbasierten SSI raus und damit entfalten die Argumente des Papers in Bezug auf die BC-Anteile der SSI ihre Wirkung. Nun können wir gern über die Argumente des Papers diskutieren (ich wäre extrem dafür, müsste es aber erstmal vollständig lesen und verstehen) und herausarbeiten, in wie fern die Argumente auf eine BC-basierte SSI zutreffen. Nur wenn wir dieses "tun", können wir auch sicherstellen, dass der moralisch/ethische Kontext intakt ist.....

**Reply by ChristophGraf-SWITCH** [reply]
@romanzoun: Was den "Level of Assurance" betrifft, sind da verschiedene Rollen im Spiel:
- Issuer: Ein Issuer mag sehr wohl Anforderungen an Wallets und Prozesse haben, die ein Holder erfüllen muss (Zertifiziertes Wallet, Biometrie, MFA, etc) und ist sonst möglicherweise nicht willens das VC auszustellen. Das würde ich im Falle des Ausstellers für E-IDs auch durchaus so erwarten
- Holder: Ich bin als Holder immer in der Pflicht selber zu entscheiden, ob ich in einem spezifischen Fall ein spezifisches Credential zeigen will oder nicht. Das kennen wir auch aus der "analogen Welt". Es gibt kaum technische Massnahmen, die es mir verunmöglichen, meine IDK in einem spezifischen Anwendungsfall "einfach so rum zeigen". Das werde ich auch in der SSI-Welt selber entscheiden müssen.

**Reply by grt-dti** [reply]
> * Holder: Ich bin als Holder immer in der Pflicht selber zu entscheiden, ob ich in einem spezifischen Fall ein spezifisches Credential zeigen will oder nicht. Das kennen wir auch aus der "analogen Welt". Es gibt kaum technische Massnahmen, die es mir verunmöglichen, meine IDK in einem spezifischen Anwendungsfall "einfach so rum zeigen". Das werde ich auch in der SSI-Welt selber entscheiden müssen.

Das ist genau der Aspekt aus dem Paper von G. Ishmaev: wie weit will man den Holder "bevormunden" (ein zu tiefer Eingriff in die Nutzungsmöglichkeiten der E-ID) oder "beschützen" (ein  willkommener Schutzes des Anwenders vor übermächtigen Verifiern) - wieviel Verantwortung will er "haben" (willkommene Freiheit) und wieviel Verantwortung muss er "tragen" (überwälzte Freiheit)?

In einer Welt mit vollständig "kognitiv konsonanten" Individuen (im Gegensatz zu "kognitiv dissonanten" Individuen), bei welchen Ursache und Wirkung immer eine Beziehung haben und wo eine wahrgenommene Freiheit immer auch eine übertragene Verantwortung zur zwingenden Folge hat (und man Verantwortlichkeit nicht einfach zu delegieren versucht), funktionieren die libertären Ideen von maximaler Freiheit - dummerweise leben wir nicht in so einer Welt und darum braucht es eine gewisse "Bevormundung" oder "Schutz" durch den Staat. Die Frage ist nur: **Wo ist die Grenze / wo soll sie sein für eine Population von 7 Mio. Internetnutzern in der heterogensten Verteilung von Wissen um Ursache/Wirkungsbeziehungen in einer digitalen / elektronischen / virtuellen Welt?**
Das ist eigentlich die moralische/ethische Fragestellung......

**Reply by romanzoun** [reply]
@ChristophGraf-SWITCH Ich würde das erweitern, um Verifier. Es muss eine Verifikation des Verifiers geben, sodass ein Nutzer nur vertrauenswürdigen Services seine Daten zeigt, zumindest gewarnt wird, wenn der Verifier ein Unbekannter ist. Ist einerseits eine Trust frage, aber auch direkt mit der Sicherheit der Nutzerdaten zutun.

**Reply by grt-dti** [reply]
> @ChristophGraf-SWITCH Ich würde das erweitern, um Verifier. Es muss eine Verifikation des Verifiers geben, sodass ein Nutzer nur vertrauenswürdigen Services seine Daten zeigt, zumindest gewarnt wird, wenn der Verifier ein Unbekannter ist. Ist einerseits eine Trust frage, aber auch direkt mit der Sicherheit der Nutzerdaten zutun.

Die Idee / das Ansinnen ist gut, birgt aber genau das Risiko, dass überreguliert wird (wie beim deutschen nPA)..... Der Grad ist schmal aber wichtig. Hier sind intelligente und vor allem adaptive / flexible Umsetzungsideen gefragt (z.B. heftig warnen aber nicht verhindern oder "Beschweren" bei einer Ombudsstelle wegen ungerechtfertigter Benachteiligung wenn die Identifikation verweigert wird, ...)

**Reply by ChristophGraf-SWITCH** [reply]
@grt-dti @romanzoun ich gehe mit euch einig, dass wir die Gratwanderung zwischen Bevormunden/Schützen und Freiheit/Selbstverantwortung gut meistern müssen und dies einen wesentlichen Erfolgsfaktor darstellen wird. Um hier zu guten, tragfähigen Lösungen zu kommen, sollten wir sicherlich mal verstehen versuchen, welche Gefahren man adressieren möchte (ich denke da mal an "Datensammelwut" und ungebührliche Profilbildung), aber auch auf welche Vertrauensanker man setzen kann: ich denke da einerseits an staatliche, regulatorische Prozesse, aber wir haben andererseits auch gut etablierte "Labels" aus allen Lebenslagen und Sektoren, die in unserem täglichen Leben in Erscheinung treten. Diese sollten wir ebenfalls einbinden und somit vom bereits bestehenden Vertrauen profitieren.
Zusammengefasst möchte ich beliebt machen die Balance auf die Seite "Freiheit" zu schieben, dies aber durch aktive Hilfestellung unter Einbezug bestehender Vertrauensanker unterstützen.

**Reply by ChristophGraf-SWITCH** [reply]
@romanzoun du hast natürlich richtigerweise erkannt, dass ich mich um die "Verifier" gedrückt habe ;-) Gerade in diesem Fall ist es wichtig, dass die Nutzer "informed decisions" machen können. Ich denke nicht, dass wir hohe Hürden für die Aufnahme von Verifiers vorsehen sollten (das wäre schlecht für Innovationen), hingegen sollte Information bereitgestellt werden, die automatisiert von User Agents ausgewertet werden kann (wie oben angesprochen die Verbindung mit bestehenden Vertrauensankern).

**Reply by grt-dti** [reply]
ich hab wenig Zweifel daran, dass wir "informed decisions" in die Köpfe der Inhaber bringen werden; Probleme wie "Fake information" stehen da natürlich auf der anderen Seite des gleichen Blattes.
Herausforderung sehe ich bei der aufgeworfenen Frage der "ungerechtfertigten Benachteiligung" wenn man nicht die vom Serviceanbieter gewünschte "informed decision" trifft. 
a. Wie kann man "Ungerechtfertigung" in einem Gesetz beschreiben? Wir können unmöglich alle gerechtfertigten Gründe ins Gesetz schreiben; oder könnte man das? Wie löst das das Ausweisgesetz - ist doch eigentlich das gleiche Problem nur mit etwas weniger Impakt?
b. Wer soll diese Benachteiligung "ahnden" und wie kann man angemessene Ahndungswerkzeuge bereitstellen, ohne das ganze E-ID System grad wieder abzuwürgen?

**Comment by grt-dti** [comment]
So; ich habs gelesen und die Problemanalyse ist an Abstraktheit fast nicht zu übertreffen - ob je jemand damit etwas brauchbares anzufangen weiss (also daraus Handlungsentscheidungen ableitet).....?
Die Analyse des Autors scheint in drei Risikorichtungen zu gehen:
- eine allzuleicht zugängliche Identität hoher (identifizierbarer) Güte in den Händen eines leicht beeinflussbaren "Holder" weckt Begehrlichkeiten staatlicher und vor allem privatwirtschaftlicher Akteure - Profiling kann man auch ohne eine singuläre Information der Identität (z.B. E-ID Nummer oder AHVN13) anstellen. Der Inhaber wird sich gegen die zumeist moralisch integer begründeten und auf der Gegenseite mit hohen zumeist persönlichen Konsequenzen behafteten Ansprüchen nur schwerlich wehren können.
- die Freiwilligkeit zum Besitz einer E-ID wird aufgrund moralisch integer begründeter Ansprüche zur Farce. Die Diskussionen zum Covid-Zertifikat sind ein Vorgeschmack auf diese Problematik
- privatwirtschaftliche Akteure (Krypto-Anarchisten) erschaffen (staatlich unkontrollierbare und unregulierbare) technologische Serviceangebote, welche "normale" staatliche Verwaltungsorganisationen nicht mehr in der "dem Impakt angemessenen Tiefe" beherrschen. Ein Staat muss dann umfassendes nicht-regulierbares und nicht-straffähiges Vertrauen in nicht-staatliche Akteure haben. Defakto hebt das den Staat aus seiner Souveränität.

Am Ende schliesst der Autor mit einer ganz einfachen Erwartung: **die Wünschbarkeit einer Identifikation muss reguliert werden** und zwar so, dass 100% der von dieser Identifikationsmöglichkeit / dem Identifikationswunsch betroffenen Individuen (also nicht nur 50,01%) dahinter stehen - Defakto eine in dieser dogmatischen Interpretation unlösbare Aufgabe.

Wie soll die allumfassende Wünschbarkeit einer Identifikation (selbst bei moralisch integeren Bedürfnissen) begrenzt werden? Wo wollen wir eine Grenze dieser Identifizierbarkeit ziehen?

**Reply by romanzoun** [reply]
sehr schwer verdaulicher Text, vielen Dank für die Zusammenfassung !

**Reply by grt-dti** [reply]
mein Text oder der Originaltext - hatte mir Mühe gegeben, das schwer verständliche Original zu vereinfachen, aber mit so vielen Querverweisen (in bester wissenschaftlicher Manier) ist es herausfordernd, die Essenz zu destillieren. Ich wage sogar zu behaupten, ich könnte den einen oder anderen Risikoaspekt übersehen haben.
Aber an der Frage der "Wunschbarkeit einer Identifikation" und der Regelung selbiger kommt man leider nicht vorbei. Was sollen sie Grenzpunkte hierbei sein?
Ein (sehr guter) Kollege vom fedpol hatte mal angedeutet, dass man sich zwingend nur gegenüber polizeistaatlichen Institutionen ausweisen/identifizieren-lassen muss......

**Reply by romanzoun** [reply]
Naja, man kann ja eine registry führen mit Services denen man vertraut und dort speichert man auch welche Daten zu sehen werden dürfen (implizierter Datenschutz)...zumindest so einen Trust Anchor plant EU mit eiDAS. 

Also das bedeutet reguliertes Vertrauensservice, dann ist die Frage ob man die Bürger alles andere verbietet oder eher auf eigene Gefahr warnt...

**Reply by grt-dti** [reply]
Halleluja; genau das ist die Frage: **Will man (in bester neoliberaler Denkweise) dem Inhaber alle Entscheide vollumfänglich selbst überlassen oder soll der Staat den Bürger/Einwohner/Inhaber vor "allzu moralisch integren" Ansinnen schützen?** 
Hierbei darf ein Aspekt nicht ausser Acht gelassen werden: Wir (also all jene "Wissende" in diesen Diskussionen hier) wissen um die negativen wie positiven Konsequenzen eines "vollständig autonomen Entscheids" für/wider E-ID Inhaberschaft und Preisgabe konkreter Attribute - das darf man nicht einfach auf eine Population von 7 Mio. übertragen......

Und um die Komplexität meiner Frage zu öffnen: genau diese Frage von G. Ishmaev hat den deutschen nPA jahrelang an einem breiteren Konsum (der nPA-ID) gehindert; die Deutschen haben den "Schutz des Bürgers vor Datenpreisgabe" zu streng reguliert und nun nachjustiert (natürlich nicht ohne vehementen Einspruch der Datenschützer und schallendem Gelächter der Nachbarn).

**Comment by pascalgottret** [comment]
Wie geht es konkret in dieser wichtigen Fragestellung weiter?
Was muss wo und in welcher Stärke reguliert werden um bei der Fragestellung rund um "Wunschbarkeit einer Identifikation" / Anti-coercion eine Antwort zu bekommen?

Anders ausgedrückt:
- Wie können wir verhindern, dass beliebige Verifier von Holdern Daten unrechtmässig einfordern, bzw erzwingen?
- Wie können wir verhindern, dass mit den eingeforderten Daten Profiling, auch z.B. durch den Staat, durchgeführt wird?

Ich sehe das Governance Framework in der Pflicht hierzu die richtigen und im Sinne des Bürgers Regulierungen vorzunehmen. Punkte davon könnten sein:
- Verifier müssen Teil eines verifizierten Ecosystems sein. Allerdings schränkt das die Verbreitung des Ecosystems massiv ein.
- Erfordernis einer nicht widerlegbaren digitalen Signatur des Verifiers für Beweisanforderungen. Damit führt der Holder im Falle eines Missbrauches ein Beweisstück.
- Einrichtung einer Ombudsstelle / Kontaktmöglichkeit einer Datenschutzstelle.
- Zero-knowledge Proof als Basisbestandteil eines Verifikationsprozesses (ausser dort, wo die Identifikation gesetzlich erforderlich ist, zB. Eröffnung eines Bankkontos)

Wie sehen die Aktivitäten rund um die Ausarbeitung des Governance Framework aus und wie sieht die Verbindung zum neuen Datenschutzgesetz per 1.9.23? Gibt es eine Arbeitstruppe, die sich mit diesen Fragestellungen beschäftigt?

**Reply by grt-dti** [reply]
gar nicht..... der aktuelle Vorschlag zum BGEID enthält effektiv wenig Vollmacht für den Bundesrat genau diese Fragen **zum Wohle** der Inhaber regulieren zu dürfen; denn dann müsste sowas sinngemässes wie "Der Bundesrat regelt die Ausführungsbestimmungen" oder "Der Bundesrat erlässt Weisung zu...." stehen und das haben wir genau im Verhältnis zum Bürger (also eigentlich genau jenem Stakeholder am Tisch um den es am Ende gehen sollte) leider eben nicht.
Es gibt genau einen "Lichtblick" in Bezug auf deine Fragestellungen: Art 10, welche dem Inhaber das "Recht" einräumt, die E-ID nur dann vorweisen zu "müssen", wenn's AwG & Co. KG verlangen, aber auch hier kann der Bundesrat "Ausnahmen vorsehen".

Dein vierter Punkt ist effektiv der ekligste Deiner Fragen ("Beweiskraft" oder "Beweiswürdigung"): Wie soll das in einem privacy by design Ansatz gehen? Wie soll man auf "unsicheren" Systemen wie einem Smartphone gerichtsverwertbare Evidenz zum Vorteil des Inhabers erzeugen? Der SSI-Ansatz läuft am Ende IMMER auf ein "Aussage gegen Aussage" Szenario hinaus, weil "Privacy by Design" die Beobachtbarkeit durch einen Dritten effektiv verhindern MUSS (genau dafür haben wir das Prinzip PbD ja). "Aussage gegen Aussage" kriegt man aber nur mitigiert, wenn's "ein Dritter beobachtet hat"......
ABER solange wir keine Beweislastumkehr kriegen, ist das alles (zum Glück) noch beherrschbar - wehe dem Fall, in einem Geschäftsumfeld würde dies mal eines Tages kommen - Dammbruch. Im ZertES hat man es versucht und (zum Glück) verhindern können - es unterliegt alles der "Beweiswürdigung durch einen Richter" aber da sind die Hürden für den Einsatz einer QES auch deutlich höher als dass, was man nun mit dem BGEID anstrebt. Wenn's Dich echt interessiert: "Änderungen anderer Erlasse" ist immer ein schönes Hilfsmittel, solche trojanischen Pferde in andere Gesetze zu schmuggeln.

**Reply by pascalgottret** [reply]
Danke für das Update. Ich werde den Punkt im Partizipationsmeeting ansprechen. Mir ist bekannt, dass aktiv am Governance Framework gearbeitet wird, aber wenn diese Punkt nicht angegangen werden haben wir ein Risiko.

> Der SSI-Ansatz läuft am Ende IMMER auf ein "Aussage gegen Aussage" Szenario hinaus, weil "Privacy by Design" die Beobachtbarkeit durch einen Dritten effektiv verhindern MUSS (genau dafür haben wir das Prinzip PbD ja). "Aussage gegen Aussage" kriegt man aber nur mitigiert, wenn's "ein Dritter beobachtet hat"......

Nicht, wenn Data Requests von Verifier an den Holder signiert werden. Dann kann der Holder beweisen, dass der Verifier spezifische Daten erfordert hat ("Non-repudiation").
Offenbar gibt es für den DID Comm Standard Ansätze dafür. Auch im Rahmen von eIDAS wird darüber diskutiert, dass diese Signatur rechtlich erzwungen werden soll.

**Comment by sspreitzer** [comment]
> Das Paper zeigt insbesondere auf, dass es imperativ ist, zur Umsetzung von SSI einen moraltheoretischen Rahmen zu schaffen der verhindert, dass damit die Büchse der Pandora geföffnet wird.

«[…] Conclusion […] We should also keep in mind that with the growing dependence of individuals on technical infrastructures, even systems designed as opt-in can quickly become de-facto necessities with the wider adoption and network effects. And any proposed ‘identity solution’ scenario should always be evaluated against with the possibility of an alternative solution that does not require any persistent identity at all. […] (Ishmaev, 2021)»

Würden wir SSI auf SBB Billette applizieren, so könne die SBB nur noch dann Billette kontrollieren, wenn es eine Kommunikationsverbindung zu SSI gäbe und somit wäre entweder ein Teil des Streckennetzes nicht kontrollierbar oder müsse zwangsweise für Kommunikation ausgebaut werden. 

Nach Ishmaev (2021) würde sich also die Frage stellen, ob dies überhaupt zweckdienlich wäre. Auf der anderen Seite kann gegenargumentiert werden, dies sei der Fortschritt und nichtmehr aufzuhaltende technologische Wandel der Digitalisierung und Industrie 4.0 (Peter et al., 2017, S. 11; Spreitzer, 2022, S. 23 ff.).

Somit auch die Frage, was ist moralisch und ethisch richtig oder falsch? In dieser Hinsicht vertrete ich die Auffassung, dass das Stimmvolk bereits die Richtung vorgegeben hat und auch, dass es das Vorhaben wieder einfangen wird, wenn es der allgemeinen Moral oder Ethik zuwiderlaufen wird.

Es bleibt also spannend.

* Ishmaev, G. Sovereignty, privacy, and ethics in blockchain-based identity management systems. Ethics Inf Technol 23, 239–252 (2021). https://doi.org/10.1007/s10676-020-09563-x
* Peter, M., Kraft, C., Eichmüller, A., & Streitengerger, M. (2017). KMU-Transformation. Als KMU
die Digitale Transformation erfolgreich umsetzen. Fachhochschule Nordwestschweiz.
* Spreitzer, S. (2022, 25. September). IT-Management als Enabler von New Work für die Unternehmensevolution [Masterarbeit, Kalaidos Fachhochschule Schweiz]. https://spreitzer.ch/publications/MBA_MA_Open_Access_Spreitzer_Sascha.pdf
